cascades-server/configuration/src/test/java/pl/gov/coi/cascades/server/persistance/hibernate/HibernateConfigurationTest.java

```diff
@@ -61,7 +63,34 @@ public void testCreateDatabaseIdGateway() throws Exception {
         );
 
         // then
-        assertThat(actual).isInstanceOf(DatabaseIdGatewayImpl.class);
+        assertThat(actual).isNotNull();
     }
 
+    @Test
+    public void testDatabaseInstanceGateway() throws Exception {
+        // given
+        HibernateConfiguration hibernateConfiguration = new HibernateConfiguration();
```

Comment:
You create HibernateConfiguration in each method. Move it out as private class property.